triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/test/LoadTesting.scala

```diff
@@ -127,6 +127,34 @@ final class BaseLoadTesting extends LoadTesting {
           succeed
         }
       }
+
+      "Contracts are created by the trigger" should {
+        "Process all contract pairings successfully" in {
```

Comment:
Testing that trigger rule iterations (and initialisation) can handle load emission